frontend/www/logout.php

```diff
@@ -1,9 +1,21 @@
 <?php
+namespace OmegaUp;
+require_once(dirname(__DIR__, 1) . '/server/bootstrap.php');
 
-require_once('../server/bootstrap.php');
 if (\OmegaUp\Controllers\Session::currentSessionAvailable()) {
     \OmegaUp\Controllers\Session::unregisterSession();
 }
 
-require_once('../server/bootstrap_smarty.php');
-$smarty->display('../templates/logout.tpl');
+\OmegaUp\UITools::render(
+    function (\OmegaUp\Request $r): array {
+        if (defined(OMEGAUP_GOOGLE_CLIENTID)) {
+            $script = ["<script src='https://apis.google.com/js/api.js' async defer></script>"];
+        }
```

Comment:
```suggestion
        $scripts = [];  // por consistencia con el nombre en L16.
        if (defined('OMEGAUP_GOOGLE_CLIENTID')) {  // esto necesita las comillas. si no, en producción esto se va a evaluar como falso! porque PHP lo va a reemplazar por `defined('619021642619-a19kvo857ei45pdh7i8ich4pf3eae3kd.apps.googleusercontent.com')`, y no hay una variable con ese nombre.
            $scripts[] = 'https://apis.google.com/js/api.js';
        }
```

así en L16 no hay necesidad del `??`.